ci-operator/step-registry/hypershift/kubevirt/run-e2e-external/hypershift-kubevirt-run-e2e-external-chain.yaml

```diff
@@ -13,6 +13,7 @@ chain:
         name: release:initial
       - env: CI_HYPERSHIFT_OPERATOR
         name: pipeline:hypershift-operator
+  - ref: hypershift-kubevirt-health-check
```

Comment:
this code should change to 
```
- ref: hypershift-kubevirt-install-hypershift
- ref: hypershift-kubevirt-health-check
```

Reply:
I altered `ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml` removing the redundant code from there.
This step is just after `  - chain: hypershift-kubevirt-create` one so I don't think I'm altering the previous flow.
Why do you think I should posticipate the check?

Reply:
okay，got it